run.sh

```diff
@@ -205,16 +205,24 @@ if [ ! -f ./gerobug_dashboard/secrets/gerobug_secret.env ]; then
     echo 'DJANGO_SUPERUSER_PASSWORD="'$DJANGO_SUPERUSER_PASSWORD'"' > ./gerobug_dashboard/secrets/gerobug_secret.env
 fi
 
+function docker_cmd() {
+    if docker --version | grep -q "version 2[0-9]\."; then
+        echo "docker compose"
+    else
+        echo "docker-compose"
+    fi
+}
+
 echo -e "\n=============================="
 echo "BUILDING GEROBUG"
 echo "=============================="
 
-docker-compose up --build --force-recreate -d
+$(docker_cmd) up --build --force-recreate --wait
 
 
 echo -e "\n=============================="
 echo "VIEWING GEROBUG LOG"
 echo "=============================="
 
 echo "(To Exit The Log Viewer, Use Ctrl + C)"
-docker-compose logs -f
\ No newline at end of file
+$(docker_cmd) logs --follow --tail 250
```

Comment:
I think long-form flags are more open-source friendly, and including the previous 250 lines could be useful for viewing startup messages if the output gets too long